include/security.php

```diff
@@ -94,6 +94,20 @@ function authenticate_success($user_record, $login_initial = false, $interactive
 
 
 	}
+
+	// If the user specified to remember the authentication, then set a cookie
+	// that expires after one week (the default is when the browser is closed).
+	// The cookie will be renewed automatically.
+	// The week ensures that sessions will expire after some inactivity.
+	if ($_SESSION['remember']) {
```

Comment:
Can you please use `isset` to check the existence of the array key, it would avoid unnecessary Notice messages when it's not set.